### DISCUSSION TITLE
Bug with the button

### DISCUSSION THREAD

**Discussion by BezrukovVik** [vuetifyjs/vuetify#20118] [General]
Hello everyone!
Finded bug with the v-btn component: if im start fast clicking LMB and RMB on button, button goes darker and darker till it become full black. Is anyone faced this bug too?
Before click:
![image](https://github.com/vuetifyjs/vuetify/assets/48985453/d4dda312-57db-4968-8b57-527ecbbe48dc)

After multiple fast LMB+RMB:
![image](https://github.com/vuetifyjs/vuetify/assets/48985453/9ef02240-2812-4358-b5f5-0ef49c74510f)

**Comment by KaelWD** [comment]
https://github.com/vuetifyjs/vuetify/issues/4185
